apis/v1/gateway_types.go

```diff
@@ -186,8 +187,8 @@ type GatewaySpec struct {
 	// +listMapKey=name
 	// +kubebuilder:validation:MinItems=1
 	// +kubebuilder:validation:MaxItems=64
-	// +kubebuilder:validation:XValidation:message="tls must be specified for protocols ['HTTPS', 'TLS']",rule="self.all(l, l.protocol in ['HTTPS', 'TLS'] ? has(l.tls) : true)"
```

Comment:
is this no longer required?

Reply:
I see is explained here https://github.com/kubernetes-sigs/gateway-api/pull/2721